project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding:utf-8 -*-
+# @ 2018 Savoir-failre Linux
+# License LGPL-3.0 or Later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models, _
+
+
+class ProjectEventWizard(models.TransientModel):
+    _name = "project.event.wizard"
+    _description = "Wizard Event Creation from Template"
+
+    template_id = fields.Many2one(
+        'event.template',
+        string='Event Template',
+    )
+    name = fields.Char(
+        string='Event Title',
+    )
+    event_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    event_notes = fields.Text(
+        string='Notes',
+    )
+    activity_line_ids = fields.One2many(
+        'activity.template.line.wiz',
+        'event_wiz_id',
+        string='Activities to Edit'
+    )
+    task_line_ids = fields.One2many(
+        'task.template.line.wiz',
+        'event_wiz_id',
+        string='Tasks to Edit'
+    )
+
+    @api.multi
+    def add_activities(self):
+        self.ensure_one()
+        template = self.env['event.template'].browse(self.template_id.id)
+        view = self.env.ref(
+            'project_event.view_project_activity_wizard')
+        for act in template.activity_template_ids:
+            activity_vals = {
+                'template_id': act.id,
+                'name': act.name,
+                'event_wiz_id': self.id,
+                'room_id': act.room_id.id,
+                'activity_resp_id': act.temp_resp_id.id,
+                'activity_category_id': act.activity_category_id.id,
+                'notes': act.notes,
+            }
+            self.env['activity.template.line.wiz'].create(activity_vals)
+
+        return {
+            'name': _('Activities Wizard'),
+            'type': 'ir.actions.act_window',
+            'view_type': 'form',
+            'view_mode': 'form',
+            'res_model': self._name,
+            'views': [(view.id, 'form')],
+            'target': 'new',
+            'res_id': self.id,
+            'context': self.env.context,
+        }
+
+    @api.multi
+    def add_tasks(self):
+        self.ensure_one()
+        activities = self.activity_line_ids
```

Comment:
@rimbendhaou Keep meaningful variable name. What will self.activity_line_ids return e.g. the list of activities or activities recordset or ....... . Please implement to other place when applicable